.yarn/plugins/@yarnpkg/plugin-typescript.cjs

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
```

Comment:
When I saw this file with `.yarnrc.yml`, these files are intended, right? Because it seems like generated one.

Reply:
Yes, it's safe to assume that everything inside .yarn is generated.